apis/Google.Storage.V1/Google.Storage.V1.Tests/UpdateBucketOptionsTest.cs

```diff
@@ -24,9 +25,10 @@ public class UpdateBucketOptionsTest
         [Fact]
         public void ModifyRequest_DefaultOptions()
         {
+            var bucket = new Bucket();
             var request = new UpdateRequest(null, null, "bucket");
```

Comment:
That's a good point - and actually, points to the fact that we don't need the parameter in the ModifyRequest method anyway. Yay.